URLQueryItemEncoder/URLQueryItemEncoder.swift

```diff
@@ -60,9 +59,9 @@ extension Array where Element == CodingKey {
     guard !isEmpty else { return "" }
     var keysPath = self
     let firstKey = keysPath.removeFirst()
-    let tailCodingKeyString = keysPath.reduce(into: "", {
+    let tailCodingKeyString = keysPath.reduce(into: "") {
       $0 += "[\($1.stringValue)]"
-    })
```

Comment:
My personal style is to use the trailing closure syntax for the control flow code. So I prefer the normal function call syntax here

Reply:
🙇